crates/uv/tests/edit.rs

```diff
@@ -889,7 +928,10 @@ fn add_remove_optional() -> Result<()> {
 
     ----- stderr -----
     warning: `uv sync` is experimental and may change without warning
-    Audited 1 package in [TIME]
+    Uninstalled 3 packages in [TIME]
+     - anyio==3.7.0
+     - idna==3.6
+     - sniffio==1.3.1
```

Comment:
A comment explaining this result would be helpful.

Reply:
No clue

Reply:
Oh, I guess `add` includes all optionals but `sync` does not.